src/write.ts

```diff
@@ -58,7 +58,9 @@ export function getJsDocString(settings: Settings, name: string, jsDoc?: JsDoc,
     if (jsDoc?.description) {
       description = getStringIndentation(jsDoc.description).deIndentedString;
     }
-    lines.push(...description.split('\n').map(line => ` * ${line}`.trimEnd()));
+    if (description) {
```

Comment:
There is an edge case with tuples I believe where the name becomes `undefined` and so does the description 🤔 I have not really been able to track it down but also I've not tried hard 🙈